src/rfbzero/experiment.py

```diff
@@ -261,6 +333,31 @@ def cycle_step(self) -> CycleStatus:
 
 
 class _ConstantVoltageCycleMode(_CycleMode):
+    """
+    Provides cycling modes (charge/discharge) for a constant voltage (CV) portion of a cycling protocol.
+
+    Parameters
+    ----------
+    charge : bool
+        True if charging, False if discharging.
+    cell_model : ZeroDModel
+        Defined cell parameters for simulating.
+    results: CyclingProtocolResults
+        Container for the simulation result data.
+    update_concentrations: callable
+        Performs coulomb counting, concentration updates via (optional) degradation and crossover mechanisms.
+    current_cutoff: float
+        Current cutoff for CV mode, below it, simulation switches from charge to discharge and vice versa (A).
```

Comment:
This line reads a bit weird to me. Maybe `...CV mode. Below it,` instead

Reply:
done